data/dataset-curation.md

```diff
@@ -50,8 +50,20 @@ sub-001
 
 Many kinds of data have a place specified for them by BIDS. See [file naming conventions](https://bids-specification.readthedocs.io/en/stable/02-common-principles.html#filesystem-structure) and the [MRI](https://bids-specification.readthedocs.io/en/stable/04-modality-specific-files/01-magnetic-resonance-imaging-data.html) and [Microscopy](https://bids-specification.readthedocs.io/en/stable/04-modality-specific-files/10-microscopy.html) extensions for full details.
 
-```{warning}
-TODO: describe neuropoly-specific BIDS entities, like bp-cspine or acq-MTon
+```{note}
+If you need to differentiate spinal cord images from the brain, use the `acq-cspine` tag. For example, `sub-001_acq-cspine_T1w.nii.gz`.
+```
+
+```{note}
+If you need to differentiate between sequences acquired with different orientations, use the `acq-axial` or `acq-sagittal` tag. For example, `sub-001_acq-axial_T1w.nii.gz`.
+```
+
+```{note}
+If you need to differentiate between different magnetization transfer (MT) seqeunces, use the `acq-MTon` or `acq-MToff` tag. For example, `sub-001_acq-MTon_T1w.nii.gz`.
```

Comment:
I think that has changed-- the new specs is indicated [here](https://bids-specification.readthedocs.io/en/stable/04-modality-specific-files/01-magnetic-resonance-imaging-data.html#anatomy-imaging-data). Example for MTS:
````
[sub](https://bids-specification.readthedocs.io/en/stable/appendices/entities.html#sub)-<[label](https://bids-specification.readthedocs.io/en/stable/appendices/entities.html#label)>[_[ses](https://bids-specification.readthedocs.io/en/stable/appendices/entities.html#ses)-<[label](https://bids-specification.readthedocs.io/en/stable/appendices/entities.html#label)>][_[acq](https://bids-specification.readthedocs.io/en/stable/appendices/entities.html#acq)-<[label](https://bids-specification.readthedocs.io/en/stable/appendices/entities.html#label)>][_[ce](https://bids-specification.readthedocs.io/en/stable/appendices/entities.html#ce)-<[label](https://bids-specification.readthedocs.io/en/stable/appendices/entities.html#label)>][_[rec](https://bids-specification.readthedocs.io/en/stable/appendices/entities.html#rec)-<[label](https://bids-specification.readthedocs.io/en/stable/appendices/entities.html#label)>][_[run](https://bids-specification.readthedocs.io/en/stable/appendices/entities.html#run)-<[index](https://bids-specification.readthedocs.io/en/stable/appendices/entities.html#index)>][_[echo](https://bids-specification.readthedocs.io/en/stable/appendices/entities.html#echo)-<[index](https://bids-specification.readthedocs.io/en/stable/appendices/entities.html#index)>]_[flip](https://bids-specification.readthedocs.io/en/stable/appendices/entities.html#flip)-<[index](https://bids-specification.readthedocs.io/en/stable/appendices/entities.html#index)>_[mt-<on|off>](https://bids-specification.readthedocs.io/en/stable/appendices/entities.html#mt)[_[part-<mag|phase|real|imag>](https://bids-specification.readthedocs.io/en/stable/appendices/entities.html#part)]_[MTS](https://bids-specification.readthedocs.io/en/stable/glossary.html#mts-suffixes)[.nii](https://bids-specification.readthedocs.io/en/stable/glossary.html#nii-extensions)[[.gz](https://bids-specification.readthedocs.io/en/stable/glossary.html#niigz-extensions)]
````

Reply:
Now it would be:

```
acq-MTon_MTS → flip-1_mt-on_MTS
acq-MToff_MTS → flip-1_mt-off_MTS
acq-T1w_MTS → flip-2_mt-off_MTS
```